src/parcel/OutputTemplate.js

```diff
@@ -12,8 +12,12 @@
 
 /* eslint-disable */
 
+const h = require('hyperscript'); // lgtm [js/unused-local-variable]
```

Comment:
hmm, I don't like that we have this dependency also for HTL...
can't we create a separate `OutputTemplate.js`  for JSX ?

Reply:
We could, but the dependency is already in `helix-pipeline`.

Reply:
but where is `h` used?

Reply:
I forgot that so far its only used in the tests. How would you pass through the output template selection?

Reply:
> I forgot that so far its only used in the tests

I don't see it :-)

Reply:
Ah, I see...it's created by parcel or babel.

Reply:
it would be better, if we could define the `h` in the script and not in the output template. I'll take a look if this is possible.

Reply:
The JSX support is pretty baked in in babel, and the examples I've seen use the annotation
```
/** @pragma h */
const h = import('hyperscript');
```

that (two-pass processing with Babel) could be an option for us.

Reply:
I find the PRAGMA computation a bit hairy in parcel:
https://github.com/parcel-bundler/parcel/blob/master/packages/core/parcel-bundler/src/transforms/babel/jsx.js

 I guess, this might cause problems if someone has a different engine in their `package.json`....

Reply:
> I guess, this might cause problems if someone has a different engine in their `package.json`…

Let's burn that bridge when we get to it.